Use beta channel for installation guide

amdprophet opened the pull request:
Add Testing Nightly Releases page

Remove carriage returns from Testing Prereleases page

Signed-off-by: Justin Kolberg <amd.prophet@gmail.com>

## Description

Changes the default installation channel to "beta".

## Motivation and Context

We're cutting the first beta today and it should become the default channel until we cut the first stable release.

Move the nightly installation instructions to a new doc.

## How Has This Been Tested?

It hasn't. The docs won't run on my system at the moment for some reason.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] New doc/guide
- [ ] Fixing errata

## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] All tests have passed.

calebhailey commented:
Saturday funday! YOLO MERGE! 

![yodel](https://cloud.githubusercontent.com/assets/141373/17834085/41765dc4-66e6-11e6-96ef-363602cf8fb7.gif)

amdprophet committed:
Use beta channel for installation guide

Add Testing Nightly Releases page

Remove carriage returns from Testing Prereleases page

Signed-off-by: Justin Kolberg <amd.prophet@gmail.com>

portertech reviewed (approved):
We still need to update the Docker instructions to use the beta image(s), but I think we should ship this.

amdprophet committed:
Merge branch 'master' into testing-betas
